ext/opcache/zend_file_cache.c

```diff
@@ -1100,6 +1100,10 @@ int zend_file_cache_script_store(zend_persistent_script *script, bool in_shm)
 	}
 #endif
 
+	if (ZCG(accel_directives).file_cache_read_only) {
+		return FAILURE;
+	}
```

Comment:
We should also avoid unlinking file caches in `zend_file_cache_script_load()`

Reply:
Thank you, I had missed these entirely - appreciate the insight!